src/platforms/android/configuration/integrations/okhttp.mdx

```diff
@@ -239,3 +239,89 @@ private final OkHttpClient client = new OkHttpClient.Builder()
   .addInterceptor(new SentryOkHttpInterceptor(new CustomBeforeSpanCallback()))
   .build();
 ```
+
+## HTTP Client errors
+
+HTTP Client errors such as bad response code are captured as error events and reported to Sentry. The error event will contain the `request` and `response` data such as `url`, `status_code`, and so on.
+
+This feature is opt-in and can be enabled by setting the `captureFailedRequests` option to `true`:
+
+```kotlin
+import okhttp3.OkHttpClient
+import io.sentry.android.okhttp.SentryOkHttpInterceptor
+
+private val client = OkHttpClient.Builder()
+  .addInterceptor(SentryOkHttpInterceptor(captureFailedRequests = true))
+  .build()
+```
+
+Only HTTP Client errors with the response code between `500` and `599` are captured as error events, but you can change this behavior by setting the `failedRequestStatusCodes` option.
+
+```kotlin
+import okhttp3.OkHttpClient
+import io.sentry.android.okhttp.SentryOkHttpInterceptor
+import io.sentry.HttpStatusCodeRange
+
+private val client = OkHttpClient.Builder()
+  .addInterceptor(SentryOkHttpInterceptor(
+    captureFailedRequests = true,
+    failedRequestStatusCodes = listOf(HttpStatusCodeRange(400, 599))))
+  .build()
+```
+
+HTTP Client errors from every target (`.*` Regular expression) will be automatically captured, but you can change this behavior by setting the `failedRequestTargets` option either with a `String.contains` value or a Regular expression.
```

Comment:
> either with a `String.contains` value

I'm a bit unclear what this means. Can we rephrase it? Does it mean we can pass a plain string, which will be checked with `String.contains`? Should we just use regular expressions and thats it, since it can also be achieved via regex?

Reply:
RFC states both, so we don't change that.
I can try to rephrase it.